frontend/front/package.json

```diff
@@ -28,7 +28,7 @@
     "web-vitals": "^2.1.0"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "PORT=3001 react-scripts start",
```

Comment:
I'm out of the loop, is this just so it'll play nicer with a local back end?

Reply:
That's right - the backend runs on port 3000.